.github/workflows/ci.yml

```diff
@@ -65,9 +65,12 @@ jobs:
         markers:
           - big
           - cli
-          - cloud
+          # `not e2e` because 1 test & 1 fixture need access token
+          - cloud and not e2e
```

Comment:
The cloud test step was always passing because everything was always being skipped due to our custom flagging system 🙃 ...
Now that we are using the `invoke ci-tests` command, it ensure the `--cloud` flag is provided and so this now runs and fails unless `not e2e` is added.